frontend/src/utils/enums.js

```diff
@@ -0,0 +1,20 @@
+export const yearsenum = {
+  FRESHMAN: 'freshman',
+  SOPHOMORE: 'sophomore',
+  JUNIOR: 'junior',
+  SENIOR: 'senior'
+}
+
+export const statusenum = {
+  ACCEPTED: 'accepted',
+  DENIED: 'rejected',
+  INTERVIEWING: 'interviewing',
+  PENDING: 'pending'
+}
+
+export const rolesenum = {
+  SWE: 'software engineer',
+  PM: 'product manager',
+  TL: 'tech lead',
+  CD: 'community director'
```

Comment:
same comment as above, where you should change these enums to match what's on the Application form.